cafe/drivers/behave/runner.py

```diff
@@ -48,40 +48,15 @@ def entry_point():
     test_env_manager = TestEnvManager(product, config)
     test_env_manager.finalize()
 
-    product_root_test_path = os.path.join(
-        test_env_manager.test_repo_path, product)
-
-    """
-    Attempts to use first positional argument after product config as a
-    sub-path to the test repo path. If not a sub-path, raise exception.
-    """
-    if behave_opts and not behave_opts[0].startswith('-'):
-        user_provided_path = behave_opts[0]
-        attempted_sub_path = os.path.join(
-            product_root_test_path,
-            user_provided_path.lstrip(os.path.sep))
-
-        if os.path.exists(attempted_sub_path):
-            behave_opts[0] = attempted_sub_path
-        else:
-            raise Exception(
-                "{directory} is not a sub-path in the {repo} repo.".format(
-                    directory=behave_opts[0],
-                    repo=test_env_manager.test_repo_package))
-    else:
-        behave_opts.insert(0, product_root_test_path)
-
-    print_mug(behave_opts[0])
+    print_mug(test_env_manager)
     behave_opts.insert(0, "behave")
 
     subprocess.call(behave_opts)
```

Comment:
I would recommend:
```
    current_directory = os.getcwd()
    product_root_test_path = os.path.join(test_env_manager.test_repo_path, product)
    os.chdir(product_root_test_path)
    subprocess.call(behave_opts)
    os.chdir(current_directory)
```

But if not, we can just cd to the location where the test is at and run the behave-runner from there.
Then the only other thing is to remove `import os` if it isn't needed.